papermill/execute.py

```diff
@@ -144,13 +146,35 @@ def prepare_notebook_metadata(nb, input_path, output_path, report_mode=False):
     return nb
 
 
+ERROR_MARKER_TAG = "papermill-error-cell-tag"
+
+ERROR_STYLE = (
+    'style="color:red; font-family:Helvetica Neue, Helvetica, Arial, sans-serif; font-size:2em;"'
+)
+
 ERROR_MESSAGE_TEMPLATE = (
-    '<span style="color:red; font-family:Helvetica Neue, Helvetica, Arial, sans-serif; font-size:2em;">'
-    "An Exception was encountered at 'In [%s]'."
+    '<span ' + ERROR_STYLE + '>'
+    "An Exception was encountered at '<a href=\"#papermill-error-cell\">In [%s]</a>'."
+    '</span>'
+)
+
+ERROR_ANCHOR_MSG = (
+    '<span id="papermill-error-cell" ' + ERROR_STYLE + '>'
+    'Execution encountered an exception here and stopped:'
```

Comment:
Nit: I would update the error text to indicate that this error was encountered when running the notebook through papermill.

Reply:
I had it mention Papermill originally, but @MSeal suggested this version in https://github.com/nteract/papermill/pull/483#discussion_r401206867

What do you both think about something like the following?

```suggestion
    'Execution using Papermill encountered an exception here and stopped:'
```

Reply:
That works imo, I'm not too partial on exact wording.

Reply:
That looks good!